docs/sources/reference/commandline/cli.md

```diff
@@ -1931,6 +1931,7 @@ To remove an image using its digest:
       -P, --publish-all=false    Publish all exposed ports to random ports
       -p, --publish=[]           Publish a container's port(s) to the host
       --pid=""                   PID namespace to use
+      --uts=""                   UTS namespace to use
```

Comment:
I think this should also be added to the "create" section.

(Unrelated, but it looks like `--pid` is missing there as well if create supports that, but that's better left for another PR)